test-results/test-results.xml

```diff
@@ -0,0 +1 @@
+<testsuites><testsuite id="1" name="Tests.Baseline" tests="2" errors="0" failures="0" timestamp="2021-12-01T08:15:26" time="0.010" hostname="SLL-DSBFMC2\SQLEXPRESS" package="tSQLt"><properties/><testcase classname="Tests.Baseline" name="test private client configuration schema exists" time="0.003"/><testcase classname="Tests.Baseline" name="test public client configuration schema exists" time="0.003"/><system-out/><system-err/></testsuite><testsuite id="2" name="Tests.MigrationHistory" tests="1" errors="0" failures="0" timestamp="2021-12-01T08:15:26" time="0.010" hostname="SLL-DSBFMC2\SQLEXPRESS" package="tSQLt"><properties/><testcase classname="Tests.MigrationHistory" name="test view exists" time="0.010"/><system-out/><system-err/></testsuite></testsuites>
```

Comment:
Should this file be committed?

Reply:
😬 no it shouldn't be, whoops!

Reply:
But hey, now you know I was testing this 😆